extensions/ql-vscode/src/vscode-tests/cli-integration/run-remote-query.test.ts

```diff
@@ -180,15 +236,37 @@ describe('Remote queries', function() {
     }
   });
 
+  function verifyQlPack(qlpackPath: string, queryPath: string, packName: string, packVersion: string) {
+    const qlPack = yaml.safeLoad(fs.readFileSync(qlpackPath, 'utf8'));
+
+    // don't check the build metadata since it is variable
+    delete (qlPack as any).buildMetadata;
+
+    expect(qlPack).to.deep.equal({
+      name: packName,
+      version: packVersion,
+      dependencies: {
+        'codeql/javascript-all': '*',
+      },
+      library: false,
+      defaultSuite: [{
+        description: 'Query suite for remote query',
+        query: queryPath,
+      }]
+    });
+  }
+
   function getFile(file: string): Uri {
     return Uri.file(path.join(baseDir, file));
   }
 
   function printDirectoryContents(dir: string) {
-    console.log(`DIR ${dir}`);
-    if (!fs.existsSync(dir)) {
-      console.log(`DIR ${dir} does not exist`);
-    }
-    fs.readdirSync(dir).sort().forEach(f => console.log(`  ${f}`));
+    dir;
```

Comment:
Do we need to worry about this `Expression has no effect` code scanning warning? 🔍

Reply:
oops...shouldn't have committed that.